docs/user/gcp/apis.md

```diff
@@ -0,0 +1,32 @@
+# GCP Service APIs
+To install OpenShift to your GCP project, certain Service APIs need to be enabled in your project *before* the openshift-installer can install the cluster
```

Comment:
to keep consistency with https://github.com/openshift/installer/pull/2515/files#diff-b2ac47237f3d701ff15f35ff95c2a961R2

```
To install OpenShift to your GCP project, the installer requires following [Service APIs][service-apis-summary] to be enabled for your project.

...

[service-apis-summary]: https://cloud.google.com/terms/services
```